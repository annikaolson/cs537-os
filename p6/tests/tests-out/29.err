Traceback (most recent call last):
  File "<string>", line 9, in <module>
FileNotFoundError: [Errno 2] No such file or directory: 'file1'
