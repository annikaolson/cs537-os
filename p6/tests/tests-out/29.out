Correct
total allocated datablocks on all disks: found 17 expected either 18 or 20.
